src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -292,10 +337,13 @@ private static string TemplateBind(IResolutionPolicy policy, PropertyInfo prop,
         {
             if (bindingData == null)
             {
+                // Skip validation if no binding data provided. We can't do the { } substitutions. 
```

Comment:
what happens in this case? an error? we just use the pattern?

we might still want to validate